### PR TITLE
Added authentication and permissions for images. Added host prefix to im...

### PR DESCRIPTION
**Pull request by jcairo**
...age URLs. Changed Image endpoint formatting. Split images endpoint into 2 seperate calls.

### CODE REVIEW

**Comment by jcairo** [comment]
Permissions for Post images are not yet set. Wasn't sure whether to use "Custom" or if we need to build one specifically for this.

File: api/views/image.py [hunk 1]
```diff
@@ -3,30 +3,54 @@
...
+    cur_dir = os.path.dirname(__file__)
+    return ('{0}/../../{1}'.format(cur_dir, img_id), mimetype)
+
+# GET /author/:authorid/image
+class AuthorImage(ListAPIView):
     """
     Returns an image.
     """
-    renderer_classes = (ImageRenderer, )
+    authentication_classes = (TokenAuthentication, BasicAuthentication, )
```
Comment (derekdowling) [inline api/views/image.py]: In place of "BasicAuthentication" use "IsEnabled" it's poorly named, but it's what checks if a user is enabled after registration and extends BasicAuth. No one will probably ever test that edge case, but just to be safe.

Comment (derekdowling) [inline api/views/image.py]: Ooops, I think this more so applies to the PostImage viewset. This one shouldn't have any because I think we want profile images to be public.

**Comment by derekdowling** [comment]
This all looks great other than the authentication class setup. Get that updated then GTG!

**Comment by jcairo** [comment]
K done.

**Comment by jcairo** [comment]
Fuck. I close commented instead of comment.
